backend/dataall/tasks/data_sharing/common/s3_approve_share.py

```diff
@@ -172,3 +172,5 @@ def get_removed_prefixes(
                     ) for prefix in removed_prefixes
                 ]
                 return removed_folders
+            return []
```

Comment:
Maybe we can remove this one and just keep the last return.  Because if no condition matches, then it will always go to the last return